### PR TITLE
documents: add other files components

### PR DESCRIPTION
**Pull request by jma**

### CODE REVIEW

**Review by PascalRepond — APPROVED** [review]

File: projects/sonar/src/app/pipe/fa-icon-class.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*
+ * RERO ILS UI
```
Comment (Garfield-fr) [inline projects/sonar/src/app/pipe/fa-icon-class.pipe.ts]: SONAR User Interface

File: projects/sonar/src/app/record/files/other-files/other-files.component.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<!--
+  RERO ILS UI
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.html]: SONAR User Interface

File: projects/sonar/src/app/record/files/other-files/other-files.component.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/*
+ * RERO ILS UI
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.scss]: SONAR User Interface

File: projects/sonar/src/app/pipe/fa-icon-class.pipe.spec.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { FaIconClassPipe } from './fa-icon-class.pipe';
```
Comment (Garfield-fr) [inline projects/sonar/src/app/pipe/fa-icon-class.pipe.spec.ts:18]: Missing header

File: projects/sonar/src/app/record/files/other-files/other-files.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.spec.ts:18]: Missing header

File: projects/sonar/src/app/record/files/other-files/other-files.component.ts [hunk 6]
```diff
@@ -0,0 +1,277 @@
+/*
+ * RERO ILS UI
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.ts]: SONAR User Interface

File: projects/sonar/src/app/record/files/stats-files/stats-files.component.spec.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/stats-files/stats-files.component.spec.ts:18]: Missing header

File: projects/sonar/src/app/record/files/stats-files/stats-files.component.ts [hunk 8]
```diff
@@ -0,0 +1,71 @@
+/*
+ * RERO ILS UI
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/stats-files/stats-files.component.ts]: SONAR User Interface

File: projects/sonar/src/app/record/files/other-files/other-files.component.html [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+            >
+              @if (file.preview) {
+              <a
+                class="file-preview"
+                href="{{ file.label }}"
+                (click)="preview(file); $event.preventDefault()"
+              >
+                <i class="fa fa-eye mx-1"></i>
+              </a>
+              } @if (file.download) {
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.html]: Return after `}`

File: projects/sonar/src/app/record/files/other-files/other-files.component.html [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+              [alt]="file.label"
+              class="file-preview w-6 shadow-1 border-300 border-1 mb-1 files-icon"
+              loading="lazy"
+            />
+          } @else {
+            <div
+            class="item-icon"
+          > <i class="text-muted fa fa-5x pb-1" [ngClass]="file.mimetype | faIconClass: 'file'"></i></div>
+          }
+            <div
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.html:69]: Indentation problem

File: projects/sonar/src/app/record/files/other-files/other-files.component.ts [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+        return 'fa-file-powerpoint-o';
+      case mimetype.startsWith('application/vnd.openxmlformats-officedocument.wordprocessingml'):
+        return 'fa-file-word-o';
+      case mimetype.startsWith('application/vnd.openxmlformats-officedocument.spreadsheetml'):
+        return 'fa-file-excel-o';
+      case mimetype.startsWith('application/pdf'):
+        return 'fa-file-pdf-o';
+    }
+    return 'fa-file-o';
+  }
```
Comment (Garfield-fr) [inline projects/sonar/src/app/record/files/other-files/other-files.component.ts]: pipe ?

**Review by Garfield-fr — APPROVED** [review]

**Pull request merged**